pcsx2-qt/Settings/AdvancedSystemSettingsWidget.cpp

```diff
@@ -33,6 +33,7 @@ AdvancedSystemSettingsWidget::AdvancedSystemSettingsWidget(SettingsDialog* dialo
 
 	SettingWidgetBinder::BindWidgetToBoolSetting(sif, m_ui.eeRecompiler, "EmuCore/CPU/Recompiler", "EnableEE", true);
 	SettingWidgetBinder::BindWidgetToBoolSetting(sif, m_ui.eeCache, "EmuCore/CPU/Recompiler", "EnableEECache", false);
+	SettingWidgetBinder::BindWidgetToBoolSetting(sif, m_ui.ee128mbram, "EmuCore/CPU/Recompiler", "EnableEE128mbRam", false);
```

Comment:
This will cause problems if it's used for per-game configurations, since the config variable will get updated, but the memory map won't reflect that. Same deal with live configuration changes. The way to work around that would be to not read the config in `CHECK_128MBRAM`, but instead some other flag latched at memory map init time.

Personally, I don't think the option should be exposed as a global setting at all, and instead only set per-game. Less risk of users breaking their setup completely then. But you'd have to solve the aforementioned memory map issue, perhaps by saving memory state, resetting, and restoring.

Reply:
> This will cause problems if it's used for per-game configurations, since the config variable will get updated, but the memory map won't reflect that. Same deal with live configuration changes. 

It won't cause problems, since the toggle is only there as a "storefront" so to say, to let users know that option exists. I specifically used it only with debugger functions, to have some "visible" changes. Another thing is external access, for example someone wants to redirect game entry point to somewhere safe, which is only after >32mb memory. He can check this option, make sure user allowed pcsx2 to use more memory and make necessary changes. With this disabled, user can have vanilla experience.

My proposal would be to make toggle greyed out with 
`m_check_EE128mbEnable->Enable(!configToApply.EnablePresets && !g_GameStarted && !g_GameLoading);`

Reply:
`!g_GameStarted && !g_GameLoading`: These are set on the emu thread, and not safe to access from the UI thread.

> He can check this option, make sure user allowed pcsx2 to use more memory and make necessary changes. With this disabled, user can have vanilla experience.

Reply:
I see, reverted. Perhaps no toggle at all then?